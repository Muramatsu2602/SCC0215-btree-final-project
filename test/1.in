1 csv/veiculo1.csv bin/veiculo1.bin

