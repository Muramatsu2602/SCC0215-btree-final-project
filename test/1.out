23269.690000
